### DISCUSSION TITLE
How to update others col and gorm.Modal DeletedAt at the same time?

### DISCUSSION THREAD

**Discussion by spencerchang** [go-gorm/gorm#5917] [Q&A]
For example:

The table struct using gorm.Modal so that DeletedAt field is auto generated.
I want to combine these two sql to one sql update. And the delete is soft deleted so only update gorm.Modal DeletedAt to a time.

db.Model(&datamodels.TableA{}).Update("Status", "off")
db.Delete(&datamodels.TableA{})

**Comment by ViniciusMarchi** [comment]
Something like that helps you?
```go
db.Model(&datamodels.TableA{}).Updates(map[string]interface{}{
    "Status": "off",
    "DeletedAt": time.Now(),
})
```

**Reply by spencerchang** [reply]
Thank you so much.
